corehq/apps/locations/views.py

```diff
@@ -265,6 +265,7 @@ def data_sources(self):
 
 
 @method_decorator(locations_access_required, name='dispatch')
+@location_safe
 class LocationsSearchView(EmwfOptionsView):
```

Comment:
Documenting here for posterity that the ancestor is already marked locations safe, and this is just a limited version of that, so it's transitively safe.

Reply:
No I think this is good to have here - I meant that with this kind of thing the PR serves as a justification for why we understand a change to be safe, and I just wanted to document that